ui/pages/confirmations/selectors/confirm.ts

```diff
@@ -68,3 +69,9 @@ export const currentSignatureRequestSecurityResponseSelector = (
 
   return state.metamask.signatureSecurityAlertResponses?.[securityAlertId];
 };
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function getIsRedesignedConfirmationsDeveloperEnabled(state: any) {
+  const { isRedesignedConfirmationsDeveloperEnabled } = getPreferences(state);
+  return isRedesignedConfirmationsDeveloperEnabled;
```

Comment:
Minor, but since we're not doing anything with the value, could we do:

`return getPreferences(state).isRedesignedConfirmationsDeveloperEnabled;`

Reply:
done